src/init.configurator.ts

```diff
@@ -0,0 +1,30 @@
+import * as fs from 'fs';
+import { Container } from 'inversify';
+import { Types } from './types';
+import { cli } from 'cli-ux';
+import { IPracticeWithMetadata } from './practices/DxPracticeDecorator';
+
+export const initialize = (scanPath: string, container: Container) => {
+  const yamlPath = `${scanPath}/.dxscannerrc.yaml`;
+  cli.action.start(`Initializing configuration: ${yamlPath}`);
+  // check if .dxscannerrc.yaml already exists
```

Comment:
Check also if there are .dxscannerrc.yml or .dxscannerrc.json. See ConfigProvider.ts

Reply:
Good catch! 👍 And the `.dxscannerrc` without an extension (for future purposes).

Reply:
I was also thinking about the file without extension but we don't check it in ConfigProvider either. So maybe we should :)

Reply:
I was thinking about this also and you're right. I checked ConfigProvider and didn't notice one condition, so we check it also there. :)